cine-aalst.py

```diff
@@ -24,21 +24,27 @@
 
 def get_movies_and_schedules():
     url = "https://cine-aalst.be/nl/movies/today"
-    response = requests.get(url)
-    content = response.text
-    pattern = r'<script id="storeData">window.storeData =(.+?);<\/script>'
-    match = re.search(pattern, content)
-
-    if match:
-        json_data = match.group(1)
-        data = json.loads(json_data)
-        movies = data.get("movies", [])
-        schedules = data.get("schedules", [])
-        cinemas = data.get("cinemas", [])
-        screens = data.get("screens", [])
-        return movies, schedules, cinemas, screens
-    else:
-        print("JSON data not found.")
+    timeout = 5  # Set a timeout in seconds (adjust as needed)
```

Comment:
These types of comments are not needed when it's very obvious what the code does.
Also, I agree on setting a timeout, especially since the default for requests is apparently `None`.
5 seconds is perhaps a bit of a low default.

```suggestion
    timeout = 10
```